packages/@eris-boiler/common/package.json

```diff
@@ -27,9 +27,9 @@
     "url": "https://github.com/alex-taxiera/eris-boiler/issues"
   },
   "peerDependencies": {
-    "eris": "^0.13.1"
+    "eris": "alex-taxiera/eris#dev"
```

Comment:
Don't depend on the direct branch. Create a tag and depend on that tag. It's bad practice.